backend/windmill-worker/src/bash_executor.rs

```diff
@@ -255,7 +257,7 @@ pub async fn handle_powershell_job(
     if !install_string.is_empty() {
         logs1.push_str("\n\nInstalling modules...");
         append_logs(&job.id, &job.workspace_id, logs1, db).await;
-        let child = Command::new("pwsh")
```

Comment:
This is a small functional change. I believe it should work but not necessarily in all scenarios which I can't test right now. Have you been able to test it?

Reply:
I've been running the changes in this patch for about ~3 weeks with script actions/flows in bash, and powershell, and python. No issues, including using some (PSGallery) external modules. This is my own community build for the workers specifically.

You're right that this is a functional change, pwsh "always" refers to "system install of powershell" and is used here specifically for installing pre-requisite modules. The other powershell invocation would be "custom build (or version) of powershell" specifically to run scripts.
I'm not a powershell expert. Having two different versions between installing dependencies (pre-script run) and for running scripts (during script run) could imply a broken script execution with a confusing message of missing dependencies. It's a distinction that can certainly be useful, but requires out-of-band setup that must overlap with configuration of the "system install of powershell", in most cases a footgun otherwise a restriction when the system repository index has (very) old powershell versions.


I should have mentioned this in the original post;
the direct reason for this PR is because I want to upstream my nix build recipe and -configuration changes for the community edition platform. The first version of the build recipe uses the <INTERPRETER>_PATH environment variables and I just continued the same structure for bash and powershell. This was never a hard requirement though, the recipe could also updates the PATH variable for all the interpreter executables. Using the <INTERPRETER>_PATH variables is more explicit, I can understand the attractiveness to that.